### DISCUSSION TITLE
New Controlling plugin: after  migration from old MR plugin amount of holidays lost

### DISCUSSION THREAD

**Discussion by idemus** [kimai/Paid-plugins#111] [Questions & Answers] [answered]
Hi Kevin,

one of my colleague has 35 days holiday this year (30 days for 2023 as of his working contract and 5 days from 2022).
After migration he has only 30 days and lost the 5 days from 2022.
Where can I correct his total amount?

Best regards
Klaus

**Comment by kevinpapst** [comment]
Holidays are not taken over from last years. You need to adjust his "holidays per year" in his work-contract settings (User profile > Edit > Working contract).

**Reply by idemus** [reply]
That solved the issue., thanks.
But it's not completely correct because the working contract has only 30 days and the other days are 'old' holiday from the last year which has nothing to do with the working contract.
So updating the amount of holiday in the working contract is not really correct.
Maybe you can change this in one of the next fixes/updates.

**Reply by kevinpapst** [reply, marked as answer]
yes, i know. it can be handled, but is far from ideal.
this is subject to change and already on the roadmap.
